source/globalCommands.py

```diff
@@ -4775,10 +4775,9 @@ def script_decreaseApplicationsVolume(self, gesture: "inputCore.InputGesture") -
 	@script(
 		description=_(
 			# Translators: Describes a command.
-			"Toggles other applications volume adjuster status",
+			"Toggle whether control of other applications' volume is enabled.",
```

Comment:
We usually use "Toggles <feature name> on and off"
```suggestion
			"Toggles application volume control on and off",
```